plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/githubPullRequest.ts

```diff
@@ -208,6 +210,16 @@ export const createPublishGithubPullRequestAction = (
             title: 'Commit Message',
             description: 'The commit message for the pull request commit',
           },
+          gitAuthorName: {
```

Comment:
I wonder if we should make these just `name` and `email` to avoid having `git` in the parameters.

@benjdlambert any thoughts here?

Reply:
Hmm we should just use the same as what we have in the other publish actions which is I think prefixed with `git`.

Reply:
Yes, name was used based gitLab.ts example